products_utils.js

```diff
@@ -19,6 +20,23 @@ const editProduct = event => {
     return productsSpecial
 }
 
+const addLocalStorageToProducts = () => {
+    let saveArrayOfProducts = []
+
+    saveArrayOfProducts = saveArrayOfProducts.concat(localStorage.getItem('products'))
+    saveArrayOfProducts = saveArrayOfProducts.toString().split(',')
+    renderedArrayProducts = saveArrayOfProducts.map(product => {
+        product = product.replaceAll('[', '').replaceAll('"', '').replaceAll(']', '')
+        productsSpecial.add(product)
+
+        return product
+    })
```

Comment:
You don't need to use let and mutate it:
```
const products = [localStorage.getItem('products').toString().split(',')]
     .map(product => product.replaceAll('[', '').replaceAll('"', '').replaceAll(']', ''))

products.forEach(product => productsSpecial.add(product))
   ```
     
Also you can check comment below and create a helper function like:

```
Storage.get('products')
```
instead of:

```
localStorage.getItem('products').toString().split(',')
```